std/traits.d

```diff
@@ -6839,3 +6839,348 @@ unittest
     struct BoolWrapper { bool value; }
     static assert(!ifTestable!(bool, a => BoolWrapper(a)));
 }
+
+/**
+ * Detect whether $(D X) is a type or not.
+ *
+ * Returns:
+ *     `bool`
```

Comment:
Why are backticks used here? shouldn't rather the DDoc macro `$(D bool)` be used ?